public/js/employee/module.js

```diff
@@ -149,7 +150,13 @@ angular.module('ShoutOut.Employee.Controllers', [])
         if ($scope.survey.questions[i].type === 'slider') {
           $scope.survey.questions[i].response = $("#question_" + i).val();
         } else if ($scope.survey.questions[i].type === 'select') {
-          $scope.survey.questions[i].response = $scope.survey.questions[i].selectedValue;
+          if ( $scope.survey.questions[i].selectedValue != undefined) {
+            $scope.survey.questions[i].response = $scope.survey.questions[i].selectedValue;
+          }
+          else {
+            $scope.survey.questions[i].response = "Not Applicable";
+          }
```

Comment:
You are right, except I changed it to "name" and not value. Because in the survey results, the legend would populate with the "value" instead of "name" and the "value" was literally just a number so it didn't really make sense.


 

    On Thursday, February 25, 2016 1:26 PM, DylanMoz <notifications@github.com> wrote:
 
 

 In public/js/employee/module.js:> @@ -149,7 +150,13 @@ angular.module('ShoutOut.Employee.Controllers', [])
>          if ($scope.survey.questions[i].type === 'slider') {
>            $scope.survey.questions[i].response = $("#question_" + i).val();
>          } else if ($scope.survey.questions[i].type === 'select') {
> -          $scope.survey.questions[i].response = $scope.survey.questions[i].selectedValue;
> +          if ( $scope.survey.questions[i].selectedValue != undefined) {
> +            $scope.survey.questions[i].response = $scope.survey.questions[i].selectedValue;
> +          }
> +          else {
> +            $scope.survey.questions[i].response = "Not Applicable";
> +          }
Just a heads up I believe this is incorrect, if its undefined (line 153), its going to assign the response to selectedValue, which is '1', '2', or '3'. So the else statement on line 157 should be assigning '3' I believe. —
Reply to this email directly or view it on GitHub.